### ISSUE TITLE
One-click install

### ISSUE THREAD

**Issue by aleclarson** [yeoman/yeoman-app#126]
Is there a simpler way to install this without the command line?

**Comment by stefanbuck** [comment]
No, unfortunately not. The app never take off the alpha state and isn't maintain anymore.

**Comment by Azerlin** [comment]
I can report that it currently doesn't work, even from the command line.
